Makefile

```diff
@@ -226,6 +230,12 @@ vanity-import-check: | $(PORTO)
 misspell: | $(MISSPELL)
 	@$(MISSPELL) -w $(ALL_DOCS)
 
+.PHONY: govulncheck
+govulncheck: | $(GOVULNCHECK)
```

Comment:
Shouldn't it be executed for each Go module like e.g. `go-mod-tidy` or `golangci-lint`?

Reply:
I was using `./...` which should execute the command in every folder recursively right?

Reply:
But let's follow the pattern which other are using, made the changes

Reply:
Why you will not test it? 😉 
The documentation says it works for a Go module.

Reply:
For example see:

```
govulncheck ./... in ./exporters/otlp/otlpmetric/otlpmetricgrpc
Scanning your code and 298 packages across 24 dependent modules for known vulnerabilities...

Vulnerability #1: GO-2023-2043
    Improper handling of special tags within script contexts in html/template
  More info: https://pkg.go.dev/vuln/GO-2023-2043
  Standard library
    Found in: html/template@go1.21
    Fixed in: html/template@go1.21.1
    Example traces found:
      #1: internal/otest/collector.go:256:30: otest.NewHTTPCollector calls http.Server.Serve, which eventually calls template.Template.Execute
      #2: internal/otest/collector.go:256:30: otest.NewHTTPCollector calls http.Server.Serve, which eventually calls template.Template.ExecuteTemplate

Vulnerability #2: GO-2023-2041
    Improper handling of HTML-like comments in script contexts in html/template
  More info: https://pkg.go.dev/vuln/GO-2023-2041
  Standard library
    Found in: html/template@go1.21
    Fixed in: html/template@go1.21.1
    Example traces found:
      #1: internal/otest/collector.go:256:30: otest.NewHTTPCollector calls http.Server.Serve, which eventually calls template.Template.Execute
      #2: internal/otest/collector.go:256:30: otest.NewHTTPCollector calls http.Server.Serve, which eventually calls template.Template.ExecuteTemplate
```

So it needs to be called for each Go module.